packages/icons-handlebars/index.js

```diff
@@ -15,15 +15,15 @@ const {
 } = require('@carbon/icon-helpers');
 const { SafeString } = require('handlebars');
 
-function iconHelper(name, { hash = {} } = {}) {
+function iconHelper(name, attributes = {}) {
```

Comment:
```suggestion
function iconHelper(name, attributes = {}, options) {
```

adding `options` would account for:

`{{iconHelper 'information' attributesObjectFromConfig another="thing" aThird="thing3"}} `